MPCAutofill/cardpicker/templates/cardpicker/base.html

```diff
@@ -18,56 +18,74 @@
         gtag('config', '{{ GTAG }}');
     </script>
 
-    <title>{% block title %} MPC Autofill {% endblock %}</title>
+    <title>{% block title %}{{ SITE_NAME }}{% endblock %}</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     {% block description %}
         <meta name="description"
               content="The easiest way to print kitchen-table proxy cards for Magic: The Gathering (MTG) with MakePlayingCards (MPC). No prior knowledge required.">
     {% endblock %}
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@4.6.0/dist/superhero/bootstrap.min.css">
+    <link rel="stylesheet" href="{{ THEME }}">
     <link rel="stylesheet" type="text/css" href="{% static 'css/custom.css' %}">
-    <script src="http://code.jquery.com/jquery.min.js"></script>
-    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui-touch-punch/0.2.3/jquery.ui.touch-punch.min.js" integrity="sha512-0bEtK0USNd96MnO4XhH8jhv3nyRF0eK87pJke6pkYf3cM0uDIhNJy9ltuzqgypoIFXw3JSuiy04tVk4AjpZdZw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <script

```

Comment:
i just updated the js & css imports for bootstrap 5 so you shouldn't need these changes :)

Reply:
Adopted your changes in the rebase so all good!